followthemoney/namespace.py

```diff
@@ -3,21 +3,24 @@
 
 Entity ID namespaces are a security mechanism related to the Aleph search index.
 
-This is because Aleph allows the user (via mappings or the API) to mint arbitary
-entity IDs. Entity IDs that are controlled by the user and not the system are,
-of course, unusual. At the same time, this makes it possible to generate bulk data
-outside Aleph, and then load entities into the system as a continuous :ref:`streams`.
+Aleph allows the user (via mappings or the API) to create arbitary entity IDs.
+Entity IDs that are controlled by the user and not the system are unusual.
+However, this makes it possible to generate bulk data outside Aleph,
+and then load entities into the system as a continuous :ref:`streams`.
 
-Namespacing works around this by making each entity ID consists of two parts:
+The problem is that having user controlled entity IDs increases the chance
+of conflict in the search index.
```

Comment:
@pudo It felt like something was missing here, so I've added this short paragraph that I hope explains it. Let me know if this isn't factually correct.